src/components/Alert/Alert.mdx

```diff
@@ -33,34 +41,18 @@ Example of using this component as a default info alert in a template
 </alert>
 ```
 
-Example of using this component as a warning alert in a template
+Example of using this component as a warning with a heading
 
 ```html
 <alert variant="warning">
+  <template v-slot:heading> Alert heading! </template>
   <p>
     You are in <span class="bold">LIVE</span> mode, all verifications will be
     charged according to your <a href="#">chosen plan</a>.
   </p>
 </alert>
 ```
 
-Example of using this component with a link
```

Comment:
nit: I wouldn't mind if we kept the link example around and just edited this snippet, but it's also not something I'm going to hold up an approval over, haha.

Reply:
sounds good, will update the example with the new link props

& yes happy to add more tests!